packages/client/src/clients/guide/client.ts

```diff
@@ -62,6 +63,18 @@ const checkForWindow = () => {
 export const guidesApiRootPath = (userId: string | undefined | null) =>
   `/v1/users/${userId}/guides`;
 
+// Detect debug params like "knock_guide_key" from URL.
+const detectDebugParams = (): DebugState => {
+  if (typeof window === "undefined") {
```

Comment:
I added the `checkForWindow` helper above recently, might be good to use it to match the pattern vs adding a new "typeof window" check? Meaning:
```
const win = checkForWindow();
if (win) { ... }
```